src/pudl/metadata/classes.py

```diff
@@ -897,11 +897,19 @@ class Contributor(PudlMeta):
         "work package leader",
     ] = "project member"
     organization: String | None = None
-    orcid: String | None = None
+    orcid: String | None | None = None
+
+    def __repr__(self):
+        """Representation excluding None values."""
+        attrs = {k: v for k, v in self.__dict__.items() if v is not None}
+        attr_str = ", ".join(f"{k}={repr(v)}" for k, v in attrs.items())
+        return f"{self.__class__.__name__}({attr_str})"
```

Comment:
I tested this and found that redefining `__repr__` doesn't affect the output of `model_dump()` and it seems like the right place for this to happen is probably over in the `pudl-archiver` repository since the serialization behavior we're trying to fix is specific to when we're creating a new `datapackage.json` for use on Zenodo.

I went ahead and tried to make [a PR that does this](https://github.com/catalyst-cooperative/pudl-archiver/pull/486)